howdoi/howdoi.py

```diff
@@ -432,6 +441,66 @@ def _get_cache_key(args):
     return str(args) + __version__
 
 
+def format_stash_item(fields, index = -1):
+    title = fields['alias']
+    description = fields['desc']
+    item_num = index + 1
+    if index == -1:
+        return '{}{}$ {}{}\n\n{}\n'.format(UNDERLINE, BOLD, title, END_FORMAT, description)
```

Comment:
I think it would be more readable if you used the following:
```
'{underline}{bold}$ {title}{end_format}\n\n{description}\n'.format(underline=UNDERLINE,
                                                                   bold=BOLD,
                                                                   title=title,
                                                                   end_format=END_FORMAT,
                                                                   description=description)
```